inc/theme-options.php

```diff
@@ -101,14 +96,6 @@ public function init_theme_settings()
         );
 
         add_settings_field(
-         'odm_news_tags',
-         __('Filter News By Tags', 'odm'),
-         array($this, 'news_tags_field'),
```

Comment:
In order for this PR to be completed, the function **news_tags_field** found on https://github.com/OpenDevelopmentMekong/wp-odm_theme/blob/master/inc/theme-options.php#L193 should also be removed, otherwise we are leaving unused code behind @S-mardii